### DISCUSSION TITLE
Laravel Livewire fails to update a component with Model Caching Enabled

### DISCUSSION THREAD

**Discussion by aj-norman** [GeneaLabs/laravel-model-caching#493] [General]
**Describe the bug**
Laravel Livewire fails to return the next/previous page of a paginated list while Model Caching is enabled. Turning model caching off or using ```disableModelCaching()``` on the query restores normal functionality

**Eloquent Query**
Please provide the complete eloquent query that caused the bug, for example:
```php
$blogs= Blog::where('user_id', '=', Auth::id())
            ->orderByDesc('date')
            ->paginate(10);
```

**Stack Trace**
No error is given by the application

**Environment**
 - PHP: 7.4
 - OS: Ubuntu 20.04
 - Laravel: 8.9
 - Livewire: 2.2.9
 - Model Caching: 0.11.0

**Additional context**

**Comment by mikebronner** [comment]
Hi @ajnorman-dev, would you be able to submit a failing unit test for this?

**Comment by aj-norman** [comment]
Hi @mikebronner, unfortunately I don't have a unit test set up any livewire component up yet. I can see that the ajax request is sent off for the second page of blogs, however the html payload returned is null.

**Comment by srirag1987** [comment]
please find a solution for Laravel Livewire pagination cache fix.

**Comment by poldixd** [comment]
Same Issue here with Livewire Pagination and the package:

```
livewire/livewire                    v2.3.6
genealabs/laravel-model-caching      0.11.1 
laravel/framework                    v8.22.0
```

**Comment by mikebronner** [comment]
@jihadismail8 Thanks for following up on this. Unfortunately I can't promise any timeline for this, as this isn't on my radar for now. I would love to review and merge a PR that addresses this, though.

**Comment by aeq-dev** [comment]
Same Issue here :/
Any updates on this please ?
